### ISSUE TITLE
View did not publish on install

### ISSUE THREAD

**Issue by MikePageDev** [Laravel-Backpack/CRUD#3405]
# Bug report

### What I did

I installed Backpack 4.1 But the dashboard view and admin controller are not there.

### What happened

I can see them in the vendor folder

### Backpack, Laravel, PHP, DB version

When I run ```php artisan backpack:version``` the output is:
```
### PHP VERSION:
PHP 7.3.24 (cli) (built: Nov 18 2020 10:14:01) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.3.24, Copyright (c) 1998-2018 Zend Technologies
    with Zend OPcache v7.3.24, Copyright (c) 1999-2018, by Zend Technologies
    with Xdebug v2.9.8, Copyright (c) 2002-2020, by Derick Rethans

### LARAVEL VERSION:
v8.19.0@f5f331cee60f1bbe672503b7eb9ba5b22b2ceacb

### BACKPACK VERSION:
4.1.29@2298dd028d6379ad93973226018d87c21aa2c6c5
```

**Comment by pxpm** [comment]
Hello @MikePageDev 

Thanks for opening your issue here in Backpack. 

Indeed, we don't publish the files, you can do it regularly with `php artisan vendor:publish` or use the built-in backpack command to publish the required file: `php artisan backpack:publish base/dashboard`.

About the controller you can use your own controller extending the default backpack controller. In a Service provider (AppServiceProvider for example), in the `register()` method, you can write something like:

```php
$this->app->bind(
        \Backpack\CRUD\app\Http\Controllers\AdminController::class,
        \App\Http\Controllers\Admin\YourCustomController::class
    );
```
Everytime APP requires backpack controller you custom controller will be served. 

I don't think this is a bug so I am going to close this. We use Github for issues or features requests, when in doubt we have other mediums where you can ask more generic questions, like this one. 

The links will be available on the BOT response.

Thanks in advance,
Pedro

**Comment by gn0rt0n** [comment]
This might be related to confusion in the documentation:
https://backpackforlaravel.com/docs/4.1/base-about#new-files-in-your-app

I had expected them to be published as well but was able to do it manually.

**Comment by pxpm** [comment]
Thanks @gn0rt0n 

I will have a look if we should publish them by default upon installation or should fix the documentation. 

Wish you the best,
Pedro

**Comment by tabacitu** [comment]
Right! Seems like those docs were out-of-date - in 4.0 `dashboard` gets published, in 4.1 `dashboard` does NOT get published. I've just updated the docs to reflect this - https://github.com/Laravel-Backpack/docs/pull/222

Thanks for bringing this to our attention @MikePageDev - it's a difficult thing to keep the docs up-to-date because for things like this we ourselves rarely look at the docs anymore, so we rely on people like you to tell us. Thanks again. Cheers!

**Comment by MikePageDev** [comment]
@tabacitu Thank you for looking into this
